gitlab/v4/objects/users.py

```diff
@@ -678,3 +680,28 @@ class UserFollowingManager(ListMixin, RESTManager):
     _path = "/users/{user_id}/following"
     _obj_cls = User
     _from_parent_attrs = {"user_id": "id"}
+
```

Comment:
In `class User()` should add an attribute `runners` that is of type `UserRunnerManager`

Then the call would be:  `user.runners.create(...)`

Reply:
@JohnVillalovos just to be sure: I think you meant `CurrentUser` (i.e. all endpoints relating to `/user/` not `/users/`).

Reply:
Ah. I think you are correct. Thanks @nejch !